CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres
+to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.0.2] - 2022-07-04
+
+### Added
+
+- Support for refunds
+- Custom User-Agent header for HTTP requests
+- Updated sandbox endpoint
+
+### Changed
+
+- Cache key for TrueLayer's client credentials token changed from `TL_SDK_AUTH-TOKEN` to `TL_CLIENT_AUTH-TOKEN`
```

Comment:
[Personal Preference] I know it was already like that on the previous release and I don't know what's the PHP standard, but I would personally prefer to just use `_` or `-` on the whole key.

Reply:
👍 I've created PHP-38 to discuss with Stefan